common/src/main/java/com/redhat/cloud/notifications/models/event/TestEventHelper.java

```diff
@@ -0,0 +1,116 @@
+package com.redhat.cloud.notifications.models.event;
+
+import com.redhat.cloud.notifications.ingress.Action;
+import com.redhat.cloud.notifications.ingress.Context;
+import com.redhat.cloud.notifications.ingress.Event;
+import com.redhat.cloud.notifications.ingress.Metadata;
+import com.redhat.cloud.notifications.ingress.Payload;
+import com.redhat.cloud.notifications.ingress.Recipient;
+
+import java.time.Clock;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+/**
+ * The purpose of this class is to help creating test events for testing the
+ * integrations that the clients have set up. The used constants are also
+ * exposed for any required checks or tests that might need them. For more
+ * information, take a look at <a href="https://issues.redhat.com/browse/RHCLOUD-22470">RHCLOUD-22470</a>.
+ */
+public class TestEventHelper {
+    /**
+     * Regular action test data.
+     */
+    public static final String TEST_ACTION_CONTEXT_ENDPOINT_ID = "test-action-context-endpoint-id";
```

Comment:
This will be seen by the org admin, let's use something a bit more user-friendly:
```suggestion
    public static final String TEST_ACTION_CONTEXT_ENDPOINT_ID = "integration-id";
```

Reply:
Renamed! I changed it to `integration-uuid` to be more precise, if that's okay!